pkg/database/realm.go

```diff
@@ -585,15 +585,20 @@ func (r *Realm) CountUsers(db *Database) (int, error) {
 }
 
 // ListUsers returns the list of users on this realm.
-func (r *Realm) ListUsers(db *Database, offset, limit int) ([]*User, error) {
+func (r *Realm) ListUsers(db *Database, offset, limit int, emailPrefix string) ([]*User, error) {
 	if limit > MaxPageSize {
 		limit = MaxPageSize
 	}
 
+	realmDB := db.db.Model(r)
+
+	if emailPrefix != "" {
+		realmDB = realmDB.Where("email like ?", fmt.Sprintf("%s%%", emailPrefix))
```

Comment:
```suggestion
		realmDB = realmDB.Where("email like %?%", fmt.Sprintf("%s", emailPrefix))
```

I also think we want to allow searching at both ends.

Reply:
We have to do it on the sprintf side, GORM puts quotes around ? and screws it up